pkg/steps/release/import_release.go

```diff
@@ -49,10 +49,11 @@ import (
 // branches of those releases.
 type importReleaseStep struct {
```

Comment:
This file contains a lot of this type of validation:
```golang
referencePolicy := imagev1.LocalTagReferencePolicy
if s.referencePolicy == imagev1.SourceTagReferencePolicy {
    referencePolicy = imagev1.SourceTagReferencePolicy
}
```
Once we change to:
```golang
ReferencePolicy *imagev1.TagReferencePolicyType `json:"reference_policy,omitempty"`
```
all this code should disappear. You will just use `s.referencePolicy` and that's it.